content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md

```diff
@@ -20,6 +20,8 @@ versions:
 {% data reusables.repositories.select-items-in-issue-or-pr-list %}
 4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items.
   ![Issues Milestone assignment drop-down](/assets/images/help/issues/issues_applying_labels_dropdown.png)
+  
+  Please note that you can also apply a label in the Labels drop-down menu within an issue or pull request.
```

Comment:
Awesome that you moved this to compliment step 4. I think we can go all the way with it and add it to step 4 😄 what do you think? 

> `4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items. You can also apply a label in the Labels drop-down menu within an issue or pull request.`
>   `![Issues Milestone assignment drop-down](/assets/images/help/issues/issues_applying_labels_dropdown.png)`
>

Reply:
Sorry, resolved it by mistake :)
Much better this way - I'll make the necessary changes.